api/src/clojure/org/akvo/flow_api/datastore/form_instance.clj

```diff
@@ -103,9 +104,14 @@
   [_ response-str]
   response-str)
 
+(def ^DateFormat sdf (SimpleDateFormat. "dd-MM-yyyy HH:mm:ss z"))
```

Comment:
Not sure if i really understand this explanation - https://clojure.org/reference/vars#set